src/sidebarLearn.json

```
@@ -25,11 +25,11 @@
       "path": "/learn/installation",
       "routes": [
         {
-          "title": "Avviare un Nuovo Progetto React",
+          "title": "Avviare un nuovo progetto React",
```

```suggestion
          "title": "Avviare un Nuovo Progetto React",
```
Teniamo la maiuscola per consistenza con la versione inglese :)